yarn.lock

```diff
@@ -5978,7 +5978,7 @@ socks@^2.6.1:
     ip "^1.1.5"
     smart-buffer "^4.1.0"
 
-source-map-support@^0.5.12, source-map-support@^0.5.16, source-map-support@^0.5.17, source-map-support@^0.5.6:
+source-map-support@^0.5.12, source-map-support@^0.5.17, source-map-support@^0.5.6:
```

Comment:
This is a bit sad - we still pull it in because of:

```
=> Found "source-map-support@0.5.21"
info Reasons this module exists
   - "skuba#ts-node-dev" depends on it
   - Hoisted from "skuba#ts-node-dev#source-map-support"
   - Hoisted from "skuba#ts-node#source-map-support"
   - Hoisted from "skuba#jest#@jest#core#jest-runner#source-map-support"
```

Fortunately this is now only a dev dep